modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/routes/Outlets/useNewProductOutlet.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {useOutletContext, useParams} from 'react-router-dom';
+import GenerateNewDXPKey from '../../../../containers/GenerateNewDXPKey/pages/GenerateNewDXPKey';
+import {PAGE_TYPES} from '../../../../utils/constants';
+
+const NewProductOutlet = () => {
+	const {productId} = useParams();
+	const {hasAccessToCurrentProduct} = useOutletContext();
+
+	const newActivationComponents = {
+		[PAGE_TYPES.dxp_new]: <GenerateNewDXPKey />,
+	};
+
+	const currentProduct = newActivationComponents[`${productId}_new`];
+
+	if (!currentProduct || !hasAccessToCurrentProduct) {
+		return <h3>Page not found2</h3>;
```

Comment:
I think this "2" shouldn't be here, right?

Reply:
Yeah, thank's for that! Done!